### PR TITLE
Add `osu!catch Convert Cup Open 2023` wiki article

### PR DESCRIPTION
**Pull request by Nivalyx**
Wiki article for the recently concluded `osu!catch Convert Cup Open 2023` tournament, on the behalf of [Urabe Mikoto](http://osu.ppy.sh/users/1272422).

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Tournaments/o!cCO/2023/en.md [hunk 1]
```diff
@@ -0,0 +1,560 @@
...
+tags:
+  - o!cCO
+  - o!cCO Season 1
+---
+
+# osu!catch Convert Cup Open 2023
+
+![osu!catch Convert Cup Open 2023 banner](img/banner.jpg)
+
+The **osu!catch Convert Cup Open 2023** (***o!cCO 2023***) was a double-elimination 2v2 osu!catch tournament hosted by ::{ flag=ID }:: [Urabe Mikoto](https://osu.ppy.sh/users/1272422) in which all the beatmaps were [converts from the osu! game mode](/wiki/Game_mode/osu!). The tournament was open to teams of osu!catch players with at least one player from Asia or Oceania in their roster. Although it was the first event to be held under the "Convert Cup Open" branding, the tournament was considered as an expansion of the previously held [Indonesia Convert Cup](/wiki/Tournaments/ICC) series.
```
Comment (TicClick) [inline wiki/Tournaments/o!cCO/2023/en.md:11]:
```suggestion
The **osu!catch Convert Cup Open 2023** (***o!cCO 2023***) was a double-elimination 2v2 osu!catch tournament hosted by ::{ flag=ID }:: [Urabe Mikoto](https://osu.ppy.sh/users/1272422) in which all the beatmaps were [converts from the osu! game mode](/wiki/Game_mode/osu!). The tournament was open to teams of osu!catch players with at least one player from Asia or Oceania in their roster. Although it was the first event to be held under the Convert Cup Open branding, the tournament was considered an expansion of the previously held [Indonesia Convert Cup](/wiki/Tournaments/ICC) series.
```

File: wiki/Tournaments/o!cCO/2023/en.md [hunk 2]
```diff
@@ -0,0 +1,560 @@
...
+- [Forum thread](https://osu.ppy.sh/community/forums/topics/1838266)
+- [Discord server](https://discord.gg/PDVF3tbhY8)
+- [Challonge brackets](https://challonge.com/OCCO2023)
+- [Livestream channel](https://www.twitch.tv/osucatchid)
+
+## Participants
+
+| Team name | Members |
+| :-: | :-- |
+| ꧁ℭ℟Åℤ¥༒₭ÏḼḼ℥℟꧂ | ::{ flag=ID }:: **[Destroid](https://osu.ppy.sh/users/5097844)**, ::{ flag=CL }:: [-Naroh](https://osu.ppy.sh/users/9087831), ::{ flag=ID }:: [EpanPan](https://osu.ppy.sh/users/13194580) |
```
Comment (TicClick) [inline wiki/Tournaments/o!cCO/2023/en.md:68]: wow

File: wiki/Tournaments/o!cCO/2023/en.md [hunk 3]
```diff
@@ -0,0 +1,560 @@
...
+   - The MixedMod pool will be played with FreeMod activated.
+   - During a MixedMod pick, one player from each team will have to play the map with the Hidden mod activated, while the other player will have to play with either HardRock or Hidden+HardRock under their belt.
+5. When playing a map from the HardRock or DoubleTime pool, the Hidden modification may be optionally used by any player.
+6. In the case of a tiebreaker, the tiebreaker map will be played with the FreeMod option enabled. Players are free to play the tiebreaker map with Hidden, HardRock, or Hidden+HardRock should they wish to.
+   - Playing the tiebreaker map with a mod is *not* mandatory.
+7. The results of each match and any other relevant information regarding the match will be noted by the referee after the match has been concluded.
+
+[^qual-note-1]: Average score across all eight Qualifier beatmaps: CTB RAIMON 1,903,488, DreStarkari 1,892,753.
+[^qual-note-2]: Average score across all eight Qualifier beatmaps: Signal Graph 1,613,901, Ban Mods 1,608,756.
+[^qual-note-3]: Average score across all eight Qualifier beatmaps: Rhodes Island 1,516,625, Ad Matai Bad 1,506,365.
```
Comment (TicClick) [inline wiki/Tournaments/o!cCO/2023/en.md:560]: what's so special about integer average ranks that you decided to add footnotes? I don't see any purpose for them to be in this article

**Review by TicClick — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/o!cCO/2023/en.md (hunk 2 above)
Comment (Nivalyx) [reply wiki/Tournaments/o!cCO/2023/en.md:68]: yeah LOL

File: wiki/Tournaments/o!cCO/2023/en.md (hunk 3 above)
Comment (Nivalyx) [reply wiki/Tournaments/o!cCO/2023/en.md:560]: this is to indicate which seed belongs to which team mainly, as several teams are tied by their average qualifier ranks

image

**Review by TicClick — APPROVED** [review]

**Pull request merged**